base/int.jl

```diff
@@ -332,6 +332,21 @@ for to in BitInteger_types, from in (BitInteger_types..., Bool)
     end
 end
 
+"""
+    rem(x::Integer, T<:Integer)
+    mod(x::Integer, T<:Integer)
+    %(x::Integer, T<:Integer)
+
+Find `y::T` such that `x` ≡ `y` (mod n), where n is the number of integers representable
+in `T`, and `y` is an integer in `[typemin(T),typemax(T)]`.
```

Comment:
Cheers either way; I lack strong feelings. (`by` sounds more natural to me, hence the suggestion.) Best!